client/app/assets/app.css

```diff
@@ -6,6 +6,8 @@ body {
     margin: 0px;
     padding: 0px;
     background-position: center;
+    background-color: #000;
+    border-radius: 10px;
```

Comment:
The border-radius wasn't being applied to the body before. Was this an intended change?

Reply:
nice catch, fixed!